btree/btree.c

```diff
@@ -2808,6 +2823,15 @@ static void ff_make(struct slot *slot)
 	/** Capture these changes in ff_capture.*/
 }
 
+static void ff_val_resize(struct slot *slot, int vsize_diff)
+{
+	struct ff_head  *h     = ff_data(slot->s_node);
```

Comment:
No. we will call node_val_resize for fixed format as well because tick function will not know the node_type, this interface will be called, but we can put the ASSERT i.e. vsize_diff == 0 , as for fixed format new value size will be same as old value size.